delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/organizations/OrganizationDeleterSpec.scala

```diff
@@ -0,0 +1,115 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.organizations
+
+import cats.syntax.all._
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.UUIDF
+import ch.epfl.bluebrain.nexus.delta.sdk.ConfigFixtures
+import ch.epfl.bluebrain.nexus.delta.sdk.generators.ProjectGen.defaultApiMappings
+import ch.epfl.bluebrain.nexus.delta.sdk.organizations.model.{Organization, OrganizationCommand}
+import ch.epfl.bluebrain.nexus.delta.sdk.organizations.model.OrganizationRejection.{OrganizationNonEmpty, OrganizationNotFound}
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.Projects.FetchOrganization
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.{ProjectsConfig, ProjectsFixture}
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.ProjectRejection.WrappedOrganizationRejection
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.{ApiMappings, ProjectFields}
+import ch.epfl.bluebrain.nexus.delta.sourcing.{GlobalEventLog, MD5}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{Identity, Label, ProjectRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.offset.Offset
+import ch.epfl.bluebrain.nexus.testkit.IOFixedClock
+import ch.epfl.bluebrain.nexus.testkit.bio.BioSuite
+import doobie.implicits._
+import doobie.util.update.Update0
+import monix.bio.{IO, Task, UIO}
+import munit.AnyFixture
+
+import java.util.UUID
+
+class OrganizationDeleterSpec extends BioSuite with IOFixedClock with ConfigFixtures {
```

Comment:
We tend to call them *Suite

Reply:
MUnit tests are *Suite, ScalaTests are *Spec

Reply:
👍